src/compressed_tensors/quantization/lifecycle/forward.py

```diff
@@ -187,37 +195,50 @@ def _process_quantization(
             scale = scale.unsqueeze(1)
             zero_point = zero_point.unsqueeze(1) if zero_point is not None else None
 
-        columns = x.shape[1]
         if columns >= group_size:
             if columns % group_size != 0:
                 raise ValueError(
-                    "tesnor column shape must be divisble "
+                    "tensor column shape must be divisble "
                     f"by the given group_size {group_size}"
                 )
-        for i in range(ceil(columns / group_size)):
-            # scale.shape should be [nchan, ndim]
-            # sc.shape should be [nchan, 1] after unsqueeze
-            sc = scale[:, i].view(-1, 1)
-            zp = zero_point[:, i].view(-1, 1) if zero_point is not None else None
 
-            idx = i * group_size
+        in_order = g_idx is None or -1 in g_idx
```

Comment:
and the proceeding logic that corresponds with actorder

Reply:
I avoided using the word `actorder` since this implementation also supports other out-of-order quantization schemes, conceivably any order that can be specified using `g_idx`.

Reply:
I see, can you add a comment saying this follow uses actorder then? Would be easier to navigate which part of the repo uses actorder flow without running it

Reply:
Gooood idea